pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -83,6 +85,65 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+
+  return (
+      <div
+          role="tabpanel"
+          hidden={props.value !== props.index}
+          id={`simple-tabpanel-${props.index}`}
+          aria-labelledby={`simple-tab-${props.index}`}
+      >
+        {props.value === props.index && (
+            <Box >
+              <Typography>{props.children}</Typography>
+            </Box>
+        )}
+      </div>
+  );
+}
+
+
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+export const ApplicationFormTabs: React.FC<ApplicationFormProps> = (props) => {
+  const [selectedTabIndex, setValue] = useState(0);
```

Comment:
```suggestion
  const [selectedTabIndex, setSelectedTabIndex] = useState(0);
```
nits, as React convention, the setter should have a name that contains the value name it controls.

Reply:
Got it!